modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/dto/v1_0/converter/UserResourceDTOConverter.java

```diff
@@ -128,6 +136,14 @@ public UserAccount toDTO(DTOConverterContext dtoConverterContext, User user)
 				setActions(dtoConverterContext::getActions);
 				setAdditionalName(user::getMiddleName);
 				setAlternateName(user::getScreenName);
+				setAssetLibraryBriefs(
+					() -> NestedFieldsSupplier.supply(
+						"assetLibraryBriefs",
+						fieldName -> TransformUtil.transformToArray(
+							user.getAllGroups(),
```

Comment:
I think we can use something like ListUtil.filter instead of having the condition inside the _toAsset*** method